hack/kubevirt-builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM kubevirt/builder@sha256:0cc33e4e2a04aa4ec44a9c38c37ff926a8b2ca67181200e31c48298bff79c4b2
+FROM kubevirt/builder@sha256:e7935549daa12e222696a2e9715e820ab6ef5b1dd0f109431856fb144928b497
```

Comment:
I can't follow here. Why are we updating the builder image when all we change there is `VERSION` in the `version.sh`?

Reply:
because I rebuilt and pushed the builder image to update bazel to something >= 1.2.0.

Reply:
That was what I was suspecting, therefore also the `.bazelversion` file for `bazelisk` now. I just found it odd that a change in bazel version was not reflected in the builder Dockerfile.

Reply:
But IIRC we agreed on just using the latest version when building new images, right?

Reply:
Maybe we should also create post commit jobs that build and push the images?

Reply:
> Maybe we should also create post commit jobs that build and push the images?

hm, the issue is that we normally need it in the PR already.